File: src/JavaLearning/LCM.java
package JavaLearning;

public class LCM {

    public static void main(String args[]) {

        int n1 = 10;
        int n2 = 4;
        int lcm= 0;

        if(n1>n2){

            lcm = n1;
        }else{

            lcm = n2;
        }


      while (true){

          if (lcm%n1==0 && lcm%2==0){

              System.out.println("The Lcm of "+n1+"and "+n2+"is "+lcm);
              break;

          }

          lcm++;
      }





    }
}

File: src/JavaLearning/LCM.kt
package JavaLearning

fun main(args: Array<String>) {

    val n1: Int = 72;
    val n2: Int = 120;
    var lcm: Int

    lcm = if (n1 > n2) n1 else n2

    while (true) {

        if (lcm % n1 == 0 && lcm % 2 == 0) {

            println("The lcm of $n1 and $n2 is $lcm")
            break
        }

        lcm++
    }


}